public/components/datasources/components/manage/manage_data_connections_table.tsx

```diff
@@ -60,30 +62,36 @@ export const ManageDataConnectionsTable = (props: HomeProps) => {
         );
       })
       .catch((err) => {
+        console.error(err);
         setToast(`Data connection $${connectionName} not deleted. See output for more details.`);
       });
   };
 
+  const fetchDataSources = () => {
+    http!
+      .get(`${DATACONNECTIONS_BASE}`)
+      .then((res: any) => {
+        const dataConnections = res.map((x: any) => {
```

Comment:
Could you add a more detailed name here for the data connection instead of x, and potentially a type for x to be?

Reply:
makes sense 👍🏽 updated PR